unit-tests/live/d400/test-auto-limits.cpp

```diff
@@ -5,7 +5,7 @@
 // This set of tests is valid for any device that supports the HDR feature //
 /////////////////////////////////////////////////////////////////////////////
 
-//#test:device D400*
+//#test:device each(D400*) !D457
```

Comment:
This will now run the test on each D400 if there are multiple (so D415, D435, etc. one after the other) -- not intended, right?